packages/shared/src/types/prices.ts

```diff
@@ -0,0 +1,17 @@
+import { CountryGroupId } from '../lib';
+
+interface ProductPriceData {
+    Monthly: {
+        price: string;
+    };
+    Annual: {
+        price: string;
+    };
+}
+interface CountryGroupPriceData {
+    GuardianWeekly: ProductPriceData;
+    Digisub: ProductPriceData;
+}
+export type Prices = {
```

Comment:
this model maps to the new scala model in support-frontend:
https://github.com/guardian/support-frontend/pull/3653/files#diff-7d440cbd5de989b27ef0b6ead72f1469f4ba9a77369d7bd639cb8dfcabc04d64R40